physics/pace/physics/stencils/physics.py

```diff
@@ -207,22 +209,18 @@ def __init__(
         )
 
         grid_indexing = stencil_factory.grid_indexing
-        origin = grid_indexing.origin_compute()
-        shape = grid_indexing.domain_full(add=(1, 1, 1))
         self._setup_statein()
         self._ptop = grid_data.ptop
         self._pktop = (self._ptop / self._p00) ** constants.KAPPA
         self._pk0inv = (1.0 / self._p00) ** constants.KAPPA
 
-        def make_storage():
-            return utils.make_storage_from_shape(
-                shape, origin=origin, backend=stencil_factory.backend
-            )
+        def make_quantity():
+            return quantity_factory.zeros(dims=[X_DIM, Y_DIM, Z_DIM], units="unknown")
 
-        self._prsik = make_storage()
-        self._dm3d = make_storage()
-        self._del_gz = make_storage()
-        self._full_zero_storage = make_storage()
+        self._prsik = make_quantity()
+        self._dm3d = make_quantity()
+        self._del_gz = make_quantity()
+        self._full_zero = make_quantity()
```

Comment:
Is this used? I did a `git grep` and couldn't find any references.

Reply:
It's unused. Removed.